### ISSUE TITLE
Reloading textures cases always index out of range excpetion

### ISSUE THREAD

**Issue by AlphaRayReal** [anegostudios/VintageStory-Issues#786]
**Game Version:** 1.14.8-rc.1 (Unstable)
**Platform:** Windows 
**Modded:** No

### Description
executed:  .reload textures
Causes index out of range exception

### How to reproduce
Crashes every time trying to reload textures

### Logs
```
Running on 64 bit Windows with 32 GB RAM 
Version: v1.14.8-rc.1 (Unstable)14.02.2021 01:15:49: Critical error occurred
System.IndexOutOfRangeException: Der Index war außerhalb des Arraybereichs.
   bei Vintagestory.Client.NoObf.ClientMain.ReloadTextures()
   bei _NsybQFqVYEcjUfBIxFNpCFRZJ4L._vKQOquTthOljtn7bZXGlP0XXSep(Int32 , CmdArgs )
   bei Vintagestory.Client.NoObf.HudDialogChat.OnNewClientToServerChatLine(Int32 groupId, String message, EnumChatType chattype, String data)
   bei Vintagestory.Client.NoObf.ClientEventManager.TriggerNewClientChatLine(Int32 groupid, String message, EnumChatType chattype, String data)
   bei Vintagestory.Client.NoObf.HudDialogChat.OnKeyDown(KeyEvent args)
   bei Vintagestory.Client.NoObf.GuiManager.OnKeyDown(KeyEvent args)
   bei Vintagestory.Client.NoObf.ClientMain.OnKeyDown(KeyEvent args)
   bei Vintagestory.Client.NoObf.ClientPlatformWindows.game_KeyDown(Object sender, KeyboardKeyEventArgs e)
   bei System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bei System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bei OpenTK.Platform.NativeWindowBase.OnKeyDown(Key key, Boolean repeat)
   bei OpenTK.Platform.Windows.WinGLNative.HandleKeyboard(IntPtr handle, WindowMessage message, IntPtr wParam, IntPtr lParam)
   bei OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wParam, IntPtr lParam)
   bei OpenTK.Platform.Windows.Functions.DispatchMessage(MSG& msg)
   bei OpenTK.Platform.Windows.WinGLNative.ProcessEvents()
   bei OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   bei _KWsZWkZAuZolPgdae7el5tHBTClA._q6fsrB8CmVl3iiVwe0lNTjMFm0(_Pz3UPCq6dLufqfMv42T2mvIRCDn , String[] )
   bei _45bJKQetuLcXXKsD9ELwUG90ErH._q6fsrB8CmVl3iiVwe0lNTjMFm0(ThreadStart )
-------------------------------
```

**Comment by radfast** [comment]
Thank you for reporting.  The dev team are aware of this issue and it will be fixed in all future 1.15 versions.  We can maybe do a partial fix for 1.14.8 so that it doesn't crash, but reloading of textures may still have other issues, e.g. it may be incomplete.

**Comment by AlphaRayReal** [comment]
> 
> 
> Thank you for reporting. The dev team are aware of this issue and it will be fixed in all future 1.15 versions. We can maybe do a partial fix for 1.14.8 so that it doesn't crash, but reloading of textures may still have other issues, e.g. it may be incomplete.

I'm actually developing first mod.  So i will change textures often. Should i downgrade to 17 or is in the next days maybe 18 rc2 out with corrected reloading?
